### DISCUSSION TITLE
Select with search and loading

### DISCUSSION THREAD

**Discussion by adamf0** [cosscom/coss#64] [Suggestions]
I would like to give a suggestion for the select component, it would be better if it could do a search and add loading to fetch data.

**Comment by davidepacilio** [comment]
Hey, @adamf0, could you please provide us with an example? A link or screenshot would be very appreciated. Thank you!

**Reply by adamf0** [reply]
like https://select2.org , but after I reopened this project the "select" part is already appropriate and can handle my case. your project is very good👍
